perk-store-webapps/src/main/webapp/WEB-INF/conf/perk-store/portal-configuration.xml

```diff
@@ -20,10 +20,10 @@
               <string>portal</string>
             </field>
             <field name="override">
-              <boolean>true</boolean>
+              <boolean>${exo.perkstore.portalConfig.metadata.override:true}</boolean>
             </field>
             <field name="importMode">
-              <string>merge</string>
+              <string>${exo.perkstore.portalConfig.metadata.importmode:merge}</string>
```

Comment:
IMO it's better to use 'insert' as default import mode as made in 6.0.x. This way, it's not re-imported when existing.

Reply:
done